### PR TITLE
Add  support for Crystal language.

### PR DESCRIPTION
**Pull request by hugopl**
Meanwhile just highlighting added.

### CODE REVIEW

**Comment by clason** [comment]
Please open a PR to the master branch; the main branch is still WIP.

File: runtime/queries/crystal/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,105 @@
+(identifier) @variable
+
+((constant) @constant
+ (#match? @constant "^[A-Z\\d_]+$"))
```
Comment (clason) [inline runtime/queries/crystal/highlights.scm:4]: Prefer `#lua-match?` for performance reasons.

File: runtime/queries/crystal/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+[(true) (false)] @boolean
+
+(nil) @constant.builtin
+
+(interpolation
+  "#{" @punctuation.special
+  "}" @punctuation.special) @embedded
+
+(comment) @comment
```
Comment (clason) [inline runtime/queries/crystal/highlights.scm:54]: `@spell`

**Comment by clason** [comment]
At least add injections for `comment`, please.

**Comment by hugopl** [comment]
I'll update it, since I wrote these queries a lot of things happened in crystal parser. Thanks for the feedback.

**Pull request closed without merging**